DietPi-Software | Docker: Log to journald + reduce log level

MichaIng opened the pull request:
**Status**: Ready

**Commit list/description**:
+ DietPi-Software | When using an invalid input argument, `dietpi-software` now prints usage info to the terminal.
+ DietPi-Software | Docker: Log to systemd-journald (journalctl) by default with reduced log level

MichaIng committed:
v6.33

+ DietPi-Software | Docker: Log to systemd-journald (journalctl) by default with reduced log level

cocoflan reviewed (commented):
Step up if you ask me!

MichaIng committed:
v6.33

+ DietPi-Software | Docker: Fix strict json syntax

MichaIng commented:
Usage/help page:
```
2020-09-27 12:44:34 root@VM-Buster:~# dietpi-software test
[  OK  ] DietPi-Software | Initialised database
[  OK  ] DietPi-Software | Reading database
[FAILED] DietPi-Software | Invalid input command (test). Aborting...

Usage: dietpi-software [<command> [<software_id>...]]
Available commands:
  <empty>                       Interactive menu to install or uninstall software
  install <software_id>...      Install each software given by space-separated list of IDs
  reinstall <software_id>...    Reinstall each software given by space-separated list of IDs
  uninstall <software_id>...    Uninstall each software given by space-separated list of IDs
  list                          Print a list with IDs and info for all available software titles
  free                          Print an unused software ID, free for a new software implementation

```
The database is initialised first, that could be fine tuned to skip it in case of invalid input.

MichaIng committed:
v6.33

+ CHANGELOG | Docker: On fresh installs or reinstalls, logs are now done with limited verbosity to systemd-journald (RAM)

MichaIng committed:
v6.33

+ DietPi-Software | Docker: Add online docs URL
